test/contrib/emscripten/test_emscripten.py

```diff
@@ -18,36 +18,29 @@
 pytest_pyodide = pytest.importorskip("pytest_pyodide")
 
 from pytest_pyodide import run_in_pyodide  # type: ignore[import-not-found] # noqa: E402
-from pytest_pyodide.decorator import (  # type: ignore[import-not-found] # noqa: E402
-    copy_files_to_pyodide,
-)
 
 from .conftest import PyodideServerInfo, ServerRunnerInfo  # noqa: E402
 
 # make our ssl certificates work in chrome
-pytest_pyodide.runner.CHROME_FLAGS.append("ignore-certificate-errors")
-
-
-# copy our wheel file to pyodide and install it
-def install_urllib3_wheel() -> (
-    typing.Callable[
-        [typing.Callable[..., typing.Any]], typing.Callable[..., typing.Any]
-    ]
-):
-    return copy_files_to_pyodide(  # type: ignore[no-any-return]
-        file_list=[("dist/*.whl", "/tmp")], install_wheels=True
+if hasattr(pytest_pyodide.runner, "CHROME_FLAGS"):
+    pytest_pyodide.runner.CHROME_FLAGS.append("ignore-certificate-errors")
+else:
+    pyodide_config = pytest_pyodide.config.get_global_config()
+    pyodide_config.set_flags(
+        "chrome", ["ignore-certificate-errors"] + pyodide_config.get_flags("chrome")
```

Comment:
Is this intended to support multiple versions of `pytest-pyodide`? Seems like you might as well pin the new version and drop the old code.